packages/financial-templates-lib/src/clients/FinancialContractEventClient.js

```diff
@@ -339,13 +339,13 @@ class ExpiringMultiPartyEventClient {
     this.firstBlockToSearch = lastBlockToSearch + 1;
 
     this.logger.debug({
-      at: "ExpiringMultiPartyEventClient",
+      at: "FinancialContractEventClient",
       message: "Expiring multi party event state updated",
```

Comment:
does this message need to change?

Reply:
good catch, done! were a few other places with the same string.